### DISCUSSION TITLE
Choose attribute shown in link child note preview

### DISCUSSION THREAD

**Discussion by nemec** [TriliumNext/Trilium#3684] [Q&A] [answered]
Is it possible to customize the content shown when linking to a child note inline within another note? In this example below it's automatically showing the `#email` attribute/label for this matching note. I'd like to be able to choose a different attribute to display here, if possible.

<img width="440" alt="2023-03-07_23-32-38" src="https://user-images.githubusercontent.com/136378/223627991-0bb50651-be49-4aeb-889e-bd4c032d2550.png">

**Comment by zadam** [comment, marked as answer]
Hi, it's not shown based on any whitelist/blacklist - this view will show any attribute which has some match against the typed autocomplete string.
